toolkits/landcover/impl/landsat8.js

```diff
@@ -71,7 +71,7 @@ Landsat8.prototype.fmaskCloudsAndShadows = function() {
   // TODO(gorelick): Is BQA in TOA calculated using Fmask as well? If so, how do
   // we model and/or abstract this for the user?
   // TODO(gino-m): Do something different or fail for type != 'SR'.
-  this.collection_ = this.collection_.map(Landsat8.applyCloudShadowBitMasks_);
+  this.collection_ = this.collection_.map(Landsat8.applyCloudShadowBitMasks);
```

Comment:
Yep. We're exposing core functionality as static methods, so thought this could be included. It also allows us to test the public API instead of a private method, which in the case of JavaScript is more of a

Sure, we can make this generic. Only thing to be aware of is static method inheritance is only supported from ES6, but since we probably don't need polymorphism here can just introduce the function it by convention. I'll do that in a follow-up PR.